### DISCUSSION TITLE
WatchOS support?

### DISCUSSION THREAD

**Discussion by MontakOleg** [swiftyfinch/Rugby#36] [Q&A] [answered]
Hi, Slava! Thanks for great tool :)

For iOS apps with embedded watchOS apps, we should build pods two times: one for iOS sdk (ouriOSAppPods) and one for watchOS sdk (ourWatchAppPods).

Is there a way to make it with Rugby?

**Comment by swiftyfinch** [comment, marked as answer]
Hey @MontakOleg, thanks for open this discussion!

Unfornatully, Rugby can't build watch sdk at this time. We need to add this feature.
Also, Rugby can't combine different SDKs for now. You can check out discussion #35 for more info.

Could you provide some public pods which support watchOS sdk for testing?

**Reply by swiftyfinch** [reply]
How many pods do you have in **ourWatchAppPods**?
Probably, if few you can just exclude it and then building in Xcode without Rugby. Try this:
```shell
rugby --exclude WatchOSPod1 WatchOSPod2
```

Rugby can build only all pods at once, without specifying **Pod-*** scheme

**Reply by MontakOleg** [reply]
>Could you provide some public pods which support watchOS sdk for testing?
>How many pods do you have in ourWatchAppPods?

We use [RxSwift](https://github.com/ReactiveX/RxSwift) in our watch app, will try your suggestion about exclude, thanks!

**Reply by MontakOleg** [reply]
Can confirm, it works with exclude 👍

**Reply by swiftyfinch** [reply]
Great news!
